base/shell/explorer/trayntfy.cpp

```diff
@@ -437,24 +649,61 @@ class CNotifyToolbar :
             StringCchCopy(notifyItem->szTip, _countof(notifyItem->szTip), iconData->szTip);
         }
 
+        if (iconData->uFlags & NIF_INFO)
+        {
+            // NOTE: In Vista+, the uTimeout value is disregarded, and the accessibility settings are used always.
+            StrNCpy(notifyItem->szInfo, iconData->szInfo, _countof(notifyItem->szInfo));
+            StrNCpy(notifyItem->szInfoTitle, iconData->szInfoTitle, _countof(notifyItem->szInfo));
+            notifyItem->dwInfoFlags = iconData->dwInfoFlags;
+            notifyItem->uTimeout = iconData->uTimeout;
+        }
+
         m_VisibleButtonCount++;
```

Comment:
It's not your code, but this should probably go to `else` branch of the `if`.

Reply:
Actually that was my code, I believe. Just from a few years ago... ;P